0.53 release work (#15181)
microsoft/PowerToys: README.md

Our [prioritized roadmap][roadmap] of features and utilities that the core team is focusing on.

#### 0.53 - December 2021 Update

We hope everyone has had a wonderful December 2021. The PowerToys community has been busy with a bunch improvements. We're still working on improving the installer but this should drastically improve things.  3 things you will want to check out are GCode support in file explorer preview pane and thumbnails, the new web search plugin from PowerToys Run via <kbd>??</kbd> action phrase, and the new Always on Top utility via <kbd>Win</kbd>+<kbd>Ctrl</kbd>+<kbd>T</kbd>.

[@Aaron-Junker](https://github.com/Aaron-Junker) also has done some great progress on implementing developer file preview via the Monaco engine from Visual Studio Code.

#### Always on Top
- With a quick <kbd>Win</kbd>+<kbd>Ctrl</kbd>+<kbd>T</kbd>, the window in focus is toggled to be on top.  Toggle again, and reverts back to normal.

#### ColorPicker

> - HEX input improvements.  Thanks @htcfreek

Reviewer: ```suggestion
- HEX input improvements for adjust color menu including support for hex code without hashtag and short hex code like #CF0.  Thanks @htcfreek
```

#### Image Resizer
- Fixed regression from Metadata tag removal of ColorSpace.  Thanks @CleanCodeDeveloper

#### PowerRename
- Row highlighting + preview support now implemented. Thanks @niels9001
- Fixed AltGR input issue
- Improved folder renaming support
- Opens on active monitor

#### PowerToys Run
- Web searching. `?? What is the answer to life` will go to your favorite search engine via your browser.  You can change the default action key too!  Thanks @cyberrex5 for primary implementation and @franky920920 and @htcfreek for supporting
- VS Code workspace improvements. Thanks @ricardosantos9521
- Binary and Hex number support.  Thanks @gsuberland
- Ability to use factorials in calculations
- PT Run will not show in Window Walker results anymore.  Thanks @davidegiacometti
- Fix log / ln calculations
- Fix to make previous results clear
- Fix to detect symlinks and prevent recursive loops
- Fix for trackpad scrolling being too fast
- Removed unneeded nuget package.  Thanks @ChaseKnowlden
- Better detection for if a packaged app can be elevated
- Improve crash resiliency for Program plugin.  Thanks @davidegiacometti

> - Improved Windows setting results.  Thanks @htcfreek

Reviewer: ```suggestion
- Improved Windows setting results for uninstalling programs.  Thanks @htcfreek
```
Do we need a note here too that translation isn't done yet?

> - Improved Windows setting results.  Thanks @htcfreek

Reviewer: ```suggestion
- Improved Windows setting results.  Thanks @htcfreek
- Fixed a bug where some similar activation phrases aren't working as expected. Thanks @htcfreek 
```

- Fixed AltGR input issue
- Improved folder renaming support
- Opens on active monitor

#### PowerToys Run
- Web searching. `?? What is the answer to life` will go to your favorite search engine via your browser.  You can change the default action key too!  Thanks @cyberrex5 for primary implementation and @franky920920 and @htcfreek for supporting
- VS Code workspace improvements. Thanks @ricardosantos9521
- Binary and Hex number support.  Thanks @gsuberland
- Ability to use factorials in calculations
- PT Run will not show in Window Walker results anymore.  Thanks @davidegiacometti
- Fix log / ln calculations
- Fix to make previous results clear
- Fix to detect symlinks and prevent recursive loops
- Fix for trackpad scrolling being too fast
- Removed unneeded nuget package.  Thanks @ChaseKnowlden
- Better detection for if a packaged app can be elevated
- Improve crash resiliency for Program plugin.  Thanks @davidegiacometti
- Improved Windows setting results.  Thanks @htcfreek
- Fixed a bug where some similar activation phrases aren't working as expected. Thanks @htcfreek and @cyberrex5.

#### Video conference mute

> - Disabled by default

Reviewer 2: ```suggestion
- Disabled by default
- Changed (default) hotkey for mute camera & microphone from <kbd>Win</kbd>+<kbd>N</kbd> to <kbd>Win</kbd>+<kbd>Shift</kbd>+<kbd>Q</kbd>
```
#15222

> We hope everyone has had a wonderful December 2021. The PowerToys community has been busy with a bunch improvements. We're still working on improving the installer but this should drastically improve things.  3 things you will want to check out are GCode support in file explorer preview pane and thumbnails, the new web search plugin from PowerToys Run via <kbd>??</kbd> action phrase, and the new Always on Top utility via <kbd>Win</kbd>+<kbd>Ctrl</kbd>+<kbd>T</kbd>.

Reviewer 3: ```suggestion
We hope everyone has had a wonderful December 2021. The PowerToys community has been busy with a bunch of improvements. We're still working on improving the installer but this should drastically improve things.  3 things you will want to check out are GCode support in file explorer preview pane and thumbnails, the new web search plugin from PowerToys Run via <kbd>??</kbd> action phrase, and the new Always on Top utility via <kbd>Win</kbd>+<kbd>Ctrl</kbd>+<kbd>T</kbd>.
```

> [@Aaron-Junker](https://github.com/Aaron-Junker) also has done some great progress on implementing developer file preview via the Monaco engine from Visual Studio Code.

Reviewer 4: We should link the issue here

> - With a quick <kbd>Win</kbd>+<kbd>Ctrl</kbd>+<kbd>T</kbd>, the window in focus is toggled to be on top.  Toggle again, and reverts back to normal.

Reviewer 2: Nit: _it_ reverts

> - With a quick <kbd>Win</kbd>+<kbd>Ctrl</kbd>+<kbd>T</kbd>, the window in focus is toggled to be on top.  Toggle again, and reverts back to normal.

Reviewer 2: Nit: _it_ reverts

> - Fix for not snapping children windows

Reviewer 2: Child windows

Our [prioritized roadmap][roadmap] of features and utilities that the core team is focusing on.

#### 0.53 - December 2021 Update

> We hope everyone has had a wonderful December 2021. The PowerToys community has been busy with a bunch of improvements. We're still working on improving the installer but this should drastically improve things.  3 things you will want to check out are GCode support in file explorer preview pane and thumbnails, the new web search plugin from PowerToys Run via <kbd>??</kbd> action phrase, and the new Always on Top utility via <kbd>Win</kbd>+<kbd>Ctrl</kbd>+<kbd>T</kbd>.

Reviewer 3: Align with phrases in issue templates.  
```suggestion
We hope everyone has had a wonderful December 2021. The PowerToys community has been busy with a bunch of improvements. We're still working on improving the installer but this should drastically improve things.  3 things you will want to check out are G-code support in file explorer preview pane and thumbnails, the new web search plugin from PowerToys Run via <kbd>??</kbd> action phrase, and the new Always on Top utility via <kbd>Win</kbd>+<kbd>Ctrl</kbd>+<kbd>T</kbd>.
```

[@Aaron-Junker](https://github.com/Aaron-Junker) also has done some great progress on [implementing developer file preview pane](https://github.com/microsoft/PowerToys/issues/1527) via the Monaco engine from Visual Studio Code.

#### Always on Top
- With a quick <kbd>Win</kbd>+<kbd>Ctrl</kbd>+<kbd>T</kbd>, the window in focus is toggled to be on top.  Toggle again, and it reverts back to normal.

#### ColorPicker

> - HEX input improvements for adjust color menu including support for hex code without hashtag and short hex code like #CF0.  Thanks @htcfreek

Reviewer 3: We do have @metions! in the 0.51's readme but not in this PR. 
```suggestion
- HEX input improvements for adjust color menu including support for hex code without hashtag and short hex code like #CF0.  Thanks @htcfreek! 
```

- Better bottom right screen detection for overlay

#### FancyZones
- Increased negative space margin
- Fix for not snapping child windows
- Fix for clearing keyboard focus on editor launch
- Fix to improve overlays to reduce brightness and hide numbers. Thanks @davidegiacometti

#### File Explorer

> - Added GCode support for thumbnails and preview pane.  Thanks @pedrolamas 

Reviewer 3: ```suggestion
- Added G-code support for thumbnails and preview pane.  Thanks @pedrolamas 
```
